react/src/Toast/useToast.ts

```diff
@@ -30,13 +30,14 @@ export type UseToastReturn = {
 
 export const useToast = ({
   status: initialStatus = 'success',
+  duration: initialDuration = 6000,
   ...initialProps
 }: UseToastProps = {}): UseToastReturn => {
   const toast = useChakraToast(initialProps)
 
   const customToastImpl = useMemo(() => {
     const impl = ({
-      duration = 6000,
+      duration = initialDuration,
```

Comment:
actually this doesn't matter right? this is just a function that we return in useTOast. the user using useToast can override the props themselves (since it is just a function) don't need this PR at all right?